Resources/doc/index.rst

```diff
@@ -348,5 +348,21 @@ class (as shown above), you can access it in the ``load()`` method.
     If you are too lazy to implement the needed method :method:`Symfony\\Component\\DependencyInjection\\ContainerInterface::setContainer`,
     you can then extend your class with :class:`Symfony\\Component\\DependencyInjection\\ContainerAware`.
 
+Automatically Generated Fixtures with Hautelook/AliceBundle
+-----------------------------------------------------------
+
+You can also use the popular `hautelook/AliceBundle`_ bundle to declare your fixtures:
+
+.. code-block:: yaml
+
+    AppBundle\Entity\Dummy:
+        dummy_{1..10}:
+            name: <name()>
+
+`hautelook/AliceBundle`_ uses `nelmio/alice`_ to load fixtures files rather than PHP class. It relies on the library `fzaninotto/Faker`_ to allow you to generate random data.
```

Comment:
why lowercase h? either composer name or namespace seems fine to me, but this looks like a mix